### ISSUE TITLE
Unicode how-much?

### ISSUE THREAD

**Issue by alerque** [starwing/luautf8#32]
From the 0.1.3 release notes:

> update Unicode version to 14

I'd be curious where you got your hands on that, it would give a lot of us a big head start! As far as I know 13 is the current version, announced on March 10th, 2020.

Also, [this package on LuaRocks.org](https://luarocks.org/modules/dannote/utf8) needs updating ;-)

**Comment by starwing** [comment]
!! I have gone to Unicode.org and just find the latest version to update... maybe I have used the test version :(

**Comment by SlySven** [comment]
> Also, [this package on LuaRocks.org](https://luarocks.org/modules/dannote/utf8) needs updating ;-)

That is true - but that Luarock was not uploaded by @starwing but @dannote, the former's one is https://luarocks.org/modules/xavier-wang/luautf8 though perhaps that might need updating as the 0.1.3-1 rock seems to be based on https://github.com/starwing/luautf8/commit/c462d19620ca4c1ec29ce02f7b322ed7d74f4ab3 but that is dated July 2020 whereas the last commit https://github.com/starwing/luautf8/commit/4e15359829f8c911de3a78ea8d4c5b945260256a was done over a year later...

**Comment by SlySven** [comment]
@starwing is it possible that you need to create and upload to https://luarocks.org/modules/xavier-wang/luautf8 a new rockspec file to support the changes that have been made over the last couple of years?

BTW The project that I code for, the MUD client [Mudlet](https://github.com/Mudlet/Mudlet.git) uses this module, we provide a Lua scripting subsystem for our end-users and we have been using (and sticking to avoid breaking backwards compatibility) on Lua 5.**1** for the last decade or so. Since we added support for more than just plain ASCII several years ago we have needed to add Unicode handling facilities and this Lua module has been perfect and been incredibly useful for that task.

:+1: Thank you.

**Comment by dannote** [comment]
@starwing Hi, I've accidentally stumbled across this thread right now. Would you like to have all the versions uploaded or just the latest one? Is there anything else I can help with?
